lerna.json

```diff
@@ -8,6 +8,7 @@
     "version": {
       "allowBranch": "master",
       "conventionalCommits": true,
+      "message": "chore(release): publish",
```

Comment:
Can we also have the published versions in the commit message?

Reply:
We can only do that in fixed versioning, message wont work for independent versions. 

> If the message contains %s, it will be replaced with the new global version version number prefixed with a "v". If the message contains %v, it will be replaced with the new global version version number without the leading "v". Note that this placeholder interpolation only applies when using the default "fixed" versioning mode, as there is no "global" version to interpolate when versioning independently.